src/games/ColorGame.js

```diff
@@ -0,0 +1,499 @@
+import React, { useState, useEffect } from 'react';
+import "../css/ColorGame.css";
+import { recordData } from "./GameDataRecorder";
+import { GAMES_ENUM } from "../constants/GamesConstants";
+
+
+
+export default function ColorGame({ advanceStateFunction }) {
+    const colors_value = [
```

Comment:
this constant, colors_names, and GameState should all also be moved outside the scope of the ColorGame component (by this I just mean put them on line 6)